File: Server-CSharp/Server-CSharp/Demo/Player/Player.cs
using Google.Protobuf;
using System;
using System.Collections.Generic;
using YT;

public class Player {
    
    //id
    public string id = "";
	public uint Conv;
	//构造函数
	public Player(uint conv){
		this.Conv = conv;
	}
	
	//数据库数据
	public PlayerData data;

	//发送信息
	public void Send(IMessage msgBase){
		Server.Clients[Conv].Send(msgBase);
	}

    


}

File: Server-Go/Demo/Player/Player/Player.go
package Player

import (
	"ReaminHistory/Demo/Player/PlayerData"
	"ReaminHistory/YT/Net"
	"google.golang.org/protobuf/runtime/protoiface"
)

type Player struct {
	//id
	Id string
	//连接号
	Conv uint32
	//数据库数据
	Data *PlayerData.PlayerData
}

func NewPlayer(conv uint32) *Player {
	return &Player{Conv: conv}
}

//发送消息
func (player *Player) Send(msg protoiface.MessageV1) {
	Net.Clients[player.Conv].Send(msg)
}
